src/common/selectors/entities.ts

```diff
@@ -9,6 +9,26 @@ export const getBuildingImages = createSelector(
 	images => images.filter((item: any) => item.build.status === 'building'),
 );
 
+export const getAllImagesCount = createSelector(
+	getImages,
+	images => images.length,
```

Comment:
it's unnecessary. Complexity of that solution is a O1, that way we don't need to use a reselect.

Reply:
use a getImages(state).length instead